Create diary.md (#19)
valentinkm/AdversarialSimulation: diary.md

> ### Previous discussion

Reviewer: would you say we should add some documentation here onto how and when to make these entries? E.g.: "After each round of collaboration in the merged simulation study, as outlined by the simulation structure, collaborators will document the process by making a diary entry. To facilitate this, a set of guiding questions is provided:"

Author: Yes! I will add it

> As you reflect on the last steps' discussion, think about the following questions:

Author: ```suggestion
After each round of collaboration in the merged simulation study, as outlined by the simulation structure, collaborators will document the process by making a diary entry. To facilitate this, a set of guiding questions is provided:

As you reflect on the last steps' discussion, think about the following questions:
```

1. Do you think you met your objectives for the past step? Explain how or why.
2. During the previous step, what kind of interactions you had with your collaborator (emails, chat, phone calls, face-to-face meetings)? Elaborate on your experience with each of these. 

> 3. How did you overcome the obstacles you saw before starting this week? Which ones are still not addressed? 

Reviewer: ```suggestion
3. How did you overcome the obstacles you saw before starting this step of collaboration? Which ones are still not addressed? 
```

4. Do you think your collaborator had influence on the work that you did individually? How or why?

### Upcoming discussion
As you reflect on the next steps' upcoming discussion, think about the following questions:

> 1. What are your group’s specific objectives for this step?

Reviewer: ```suggestion
1. What are your specific objectives for this step?
```
